gd/gd.php

```diff
@@ -223,19 +224,37 @@ function imagecolorallocate ($image, $red, $green, $blue) {}
  */
 function imagepalettecopy ($destination, $source) {}
 
+#[PhpStormStubsElementAvailable("8.0")]
 /**
  * Create a new image from the image stream in the string
  * @link https://php.net/manual/en/function.imagecreatefromstring.php
  * @param string $image <p>
  * A string containing the image data.
  * </p>
- * @return resource|GdImage|false An image resource will be returned on success. <b>FALSE</b> is returned if
+ * @return GdImage|false An image resource will be returned on success. <b>FALSE</b> is returned if
  * the image type is unsupported, the data is not in a recognised format,
  * or the image is corrupt and cannot be loaded.
  */
 #[Pure]
 function imagecreatefromstring ($image) {}
 
+#[PhpStormStubsElementAvailable('5.3','7.4')]
```

Comment:
As the matter of code style I'd group multiple attributes together

Reply:
I thought about it but decided to insert the annotation before the PHPDoc for better readability but this is discussable I guess and depends on the navigation patterns:) If GoTo is used, it's better to have attributes together if scrolling - better to have a new attribute above.